src/components/elementView/rackSelectView.tsx

```diff
@@ -69,39 +63,10 @@ class RackSelectView extends React.Component<
     const headers = getHeaders(this.props.token);
     this.props.submitForm(this.state.values, headers);
   };
-  renderRackOptions(range: boolean) { }
-  componentDidMount() { }
```

Comment:
Just to be sure, was this whole section removed because we no longer need to get Datacenters, as they are part of the shared state even for racks?

Reply:
rackselectview basically only handles allowing users to input a rack range, so the datacenter that the rack range is for is handled at a higher level component